src/todo/ToDoContainer.js

```diff
@@ -1,19 +1,19 @@
 import React, { useState } from 'react';
-import ListListOfItems from './ToDoList';
-import InputAndBtn from './CreateTodo';
+import CreateTodo from './CreateTodo';
+import TodoList from './ToDoList';
 
-function Todo(){
+function TodoContainer(){
     const [initialText, setInitialText] = useState('');
```

Comment:
You don't need this state anymore. You can retrieve the text value from your callback